pyp5js/tests/http/test_web_app.py

```diff
@@ -68,7 +68,7 @@ def test_post_with_existing_sketch_name_should_render_form_with_error(self):
         self.create_sketch('my_existing_sketch')
         self.client.post(self.route, data={'sketch_name': 'my_existing_sketch'})
         self.assert_template_used('new_sketch_form.html')
-        self.assert_context('error', 'The sketch tests-sketchbook/my_existing_sketch already exists.')
+        self.assert_context('error', 'The sketch ' + os.path.join('tests-sketchbook', 'my_existing_sketch') + ' already exists.')
```

Comment:
Can we assing the `os.path.join` to a variable and use a f-string in the assertion?